Update README.md

antonio-soares-altium opened the pull request:
Updated description of toolbox installation and management.

Deleted old Matlab App installation file and included new Toolbox installation file and updated Matlab Toolbox Project file.

antonio-soares-altium committed:
Updated Toolbox to new extension

Updated the Toolbox project file, created the Toolbox with new extension, deleted old Matlab App extension (incompatible with new version definition of Apps) and updated README text.

antonio-soares-altium committed:
Update README.md

Updated description of toolbox installation and management.

antonio-soares-altium committed:
Add files via upload

Updated toolbox project file and toolbox file compatible with newer versions.

antonio-soares-altium committed:
Delete ValispaceMatlabToolbox.mlappinstall

Old Matlab App installation file which won't install toolbox on newer version.

antonio-soares-altium committed:
Merge branch 'version-compatibility-updates' of https://github.com/valispace/ValispaceMatlabToolbox into version-compatibility-updates

louise-lindblad-altium reviewed (dismissed):
Looks good!

antonio-soares-altium committed:
Changed login to Basic Auth

Since OAUTH was broken the Toolbox was unusable. Changes were done to ValispaceInit() so that it uses standard auth. Version a 1.36.10

antonio-soares-altium committed:
Update README.md

Updated version for the latest modification

antonio-soares-altium committed:
Update ValispaceInit.m

First attempt to change it to token auth

antonio-soares-altium committed:
ValispaceInit.m updated

updated to corrected for authHeader error

antonio-soares-altium committed:
updated ValispaceInit.m

part of the ChatGPT generated code was not working so it got commented out (line 48).
Basic functionality restored using authentication token.

antonio-soares-altium reviewed (commented):
Basic commands tested and working

antonio-perez-altium commented on src/ValispaceInit.m:
```diff
@@ -1,36 +1,86 @@
+
+    %% Step 2: Check for SSL Connection
+    if startsWith(URL, 'http://', 'IgnoreCase', true)
+        if nargin == 3 && strcmpi(insecure, 'insecure')
+            warning(['You are sending your credentials over an unencrypted connection. ' ...
+                     'It is highly recommended to use "https://..." instead!']);
         else
-            error('VALISPACE-ERROR: You are trying to send your password-credentials using an unencrypted connection. Better use "https://..." instead! If you still want to go ahead with an insecure connection, please use ValisapceInit() with "insecure" as a last argument.');
+            error(['VALISPACE-ERROR: You are attempting to send credentials over an unencrypted ' ...
+                   'connection. Please use "https://..." for secure communication. ' ...
+                   'If you intend to proceed with an insecure connection, use ValispaceInit() ' ...
+                   'with "insecure" as the last argument.']);
         end
     end
```
@joao-goncalves-altium should we allow insecure connections to be established with our integrations??

antonio-perez-altium commented on ValispaceMatlabToolbox.prj:
```diff
@@ -0,0 +1,173 @@
+
+Version
+The current version of the Matlab toolbox is 1.3.7</param.description>
+    <param.screenshot>C:\Users\anton.VALISPACE-SM-AS\OneDrive - Altium LLC\Pictures\Valispace\Valispace Logo.png</param.screenshot>
```
Exposing local path should not be done, please input reference in another way, either a relative file in the repo if possible or to a remote url

antonio-perez-altium reviewed (approved):
As the majority of the dev team works now on Mac/Unix, I would suggest adding a warning on the README that this `.prj` file only supports Windows for development.

antonio-perez-altium reviewed (changes requested):
Meant to request changes

antonio-soares-altium committed:
Updated the Logo and source

Updated the logo to use A365 logo and added the file to the root of the repo so now the filepath is relative as suggested. Also updated the toolbox installation file with the new logo and description.

antonio-soares-altium committed:
unsecure login option removed
